DEHPEcosimPro.Tests/DstController/DstControllerTestFixture.cs

```diff
@@ -705,11 +710,11 @@ public void VerifyWhenConnectionToOpcStatus()
 
             this.mappingConfigurationService.Verify(
                 x => x.LoadMappingFromDstToHub(It.IsAny<ReactiveList<VariableRowViewModel>>()),
-                Times.Exactly(2));
+                Times.Exactly(3));
```

Comment:
If you add that one time where the OpcClientStatusCode is ErrorAddSubscribption, why then the mapping gets (re)loaded?